### ISSUE TITLE
Shield Slam: KeyError in managers.py

### ISSUE THREAD

**Issue by beheh** [jleclanche/fireplace#105]
```
DEBUG:root:<Spell ('Shield Slam')> moves from <Zone.HAND: 3> to <Zone.PLAY: 1>
INFO:root:Activating <Spell ('Shield Slam')> action targeting <Minion ('Ancient Watcher')>
INFO:root:<Spell ('Shield Slam')> triggering <TargetedAction: Hit(AMOUNT=Attr(<Selector: PLAY FRIENDLY and HERO and>, <GameTag.ARMOR: 292>))> targeting [<Minion ('Ancient Watcher')>]
Traceback (most recent call last):
  File "./full_game.py", line 52, in <module>
    main()
  File "./full_game.py", line 37, in main
    card.play(target=random.choice(card.targets))
  File "../fireplace/card.py", line 312, in play
    self.game.queue_actions(self.controller, [Play(self, target, choose)])
  File "../fireplace/game.py", line 201, in queue_actions
    ret.append(action.trigger(source))
  File "../fireplace/actions.py", line 108, in trigger
    self.do(source, *args)
  File "../fireplace/actions.py", line 231, in do
    card.action()
  File "../fireplace/card.py", line 224, in action
    self.game.queue_actions(self, actions)
  File "../fireplace/game.py", line 201, in queue_actions
    ret.append(action.trigger(source))
  File "../fireplace/actions.py", line 313, in trigger
    target_args = self.get_target_args(source, target)
  File "../fireplace/actions.py", line 282, in get_target_args
    v = v.evaluate(source)
  File "../fireplace/dsl/lazynum.py", line 70, in evaluate
    return sum(e.tags[self.tag] for e in entities)
  File "../fireplace/dsl/lazynum.py", line 70, in <genexpr>
    return sum(e.tags[self.tag] for e in entities)
  File "../fireplace/managers.py", line 12, in __getitem__
    raise KeyError
KeyError
```

Test case:
```python
def test_shield_slam():
	game = prepare_game(WARRIOR, WARRIOR)
	wisp = game.player1.summon(WISP)
	assert game.player1.hero.armor == 0
	game.player1.give("EX1_410").play(target=wisp)
	assert not wisp.dead
	game.player1.hero.power.use()
	assert game.player1.hero.armor == 2
	game.player1.give("EX1_410").play(target=wisp)
	assert wisp.dead
```

**Comment by beheh** [comment]
The error occurs when Shield Slam tries to access the armor tag.

**Comment by jleclanche** [comment]
Fixed by 1f2b5ae0a21c8a0587d76aaedae5e1ce44e8dcd0
